<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>784</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E88E5 </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_imagenUat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Recursos.qrc">:/Logos/Archivos/log_uat_nuevo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenFi">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Recursos.qrc">:/Logos/Archivos/FIT_logo_vertical.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenGatos">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Recursos.qrc">:/Logos/Archivos/export202502092301563482.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Simular_Semaforo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>510</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>Simular Sem치foro</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Simular un Sem치foro</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Unidad2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Unidad 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenApagado">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>171</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Recursos.qrc">:/Ejercicios/Archivos/Apagago.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Color">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>El sem치foro est치 en:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Tiempo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>650</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Espere:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Tiempo_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Avance:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
